sirepo/template/raydata.py

```diff
@@ -118,6 +137,10 @@ def write_parameters(data, run_dir, is_parallel):
             d.join(f).write_binary(b)
 
 
+def _catalog():
+    return databroker.catalog[_CATALOG_NAME]
```

Comment:
Perhaps wrap in a PKDict so ```scan_info()``` can use ```owner=m.start.owner``` etc.

Reply:
I'm hesitant to do that. Between Bluesky and Intake there is a lot of manipulation of underlying datastructures to make things look like a dict that aren't actually a dict (ex a mongoDB). So I'm concerned that things will go strange (ex pull in all of the data when things would otherwise normally be lazily loaded). @robnagler do you have thoughts?

Reply:
On closer inspection of the PKDict code I guess this won't happen. It ends up just calling the dict like normal. But nested structures may prove challenging. I need to test.

Reply:
Correct. A catalog is almost certainly not a dict, and instead it implements `__getitem__`.

Reply:
It's a mere convenience in a single place, so not worth much effort to disentangle I guess. It's acting like a dict in scan_info, which prompted my comment.